build/node.dev.Dockerfile

```diff
@@ -28,6 +28,7 @@ RUN mkdir -p /go/src/github.com/eagraf/habitat-new/pkg
 EXPOSE 3000
 EXPOSE 3001
 EXPOSE 4000
+EXPOSE 80
```

Comment:
is this for `tailscale`? I wonder if we can specify different ports

Reply:
I think this is leftover from when I was testing